flutter/lib/src/event_processor/flutter_enricher_event_processor.dart

```diff
@@ -237,4 +243,13 @@ class FlutterEnricherEventProcessor implements EventProcessor {
       inForeground: inForeground,
     );
   }
+
+  void _getAppScreen(Contexts contexts, SentryApp app) {
```

Comment:
let's rename this method to something more explicit, like `_updateAppCurrentRoute`. wdyt?